src/rp2_common/pico_cyw43_arch/include/pico/cyw43_arch.h

```diff
@@ -427,7 +431,9 @@ int cyw43_arch_wifi_connect_bssid_blocking(const char *ssid, const uint8_t *bssi
  *             \ref CYW43_AUTH_WPA2_AES_PSK, or \ref CYW43_AUTH_WPA2_MIXED_PSK (see \ref CYW43_AUTH_)
  * \param timeout how long to wait in milliseconds for a connection to succeed before giving up
  *
- * \return 0 if the initialization is successful, an error code otherwise see \ref pico_error_codes
+ * \return 0 if the connection is successful. PICO_ERROR_TIMEOUT is returned if the timeout is reached before a successful connection.
```

Comment:
```suggestion
 * \return 0 if the connection is successful.
 * PICO_ERROR_TIMEOUT is returned if the timeout is reached before a successful connection.
```
For consistency with the way you list the other errors.